### DISCUSSION TITLE
Embedding lua in v

### DISCUSSION THREAD

**Discussion by GuilHartt** [vlang/v#18080] [Questions and Answers] [answered]
Hello everyone, can anyone tell me if there is any lua binding for v? I looked everywhere but couldn't find it.

**Comment by hungrybluedev** [comment]
There isn't one yet. Perhaps you can try using the C API from here: <https://www.lua.org/pil/24.html>

First try an get lua to work in C. Then try following this tutorial: <https://blog.vosca.dev/c-from-v-1/>

If you do it successfully, try making a V wrapper for it! Then everyone can use it and contribute to it.

**Comment by roberto170** [comment, marked as answer]
here : https://github.com/roberto170/embedd-lua-in-vlang/tree/main
